src/meta/src/hummock/manager/mod.rs

```diff
@@ -2509,6 +2530,18 @@ impl HummockManager {
                 }
                 let state_table_size = *table_size;
 
+                {
+                    if is_high_write_throughput {
+                        table_vnode_partition_mapping.insert(*table_id, 8_u32);
+                    } else if state_table_size > self.env.opts.cut_table_size_limit {
+                        table_vnode_partition_mapping.insert(*table_id, 4_u32);
+                    }
+                }
+
+                if !created_tables.contains(table_id) {
```

Comment:
Why？ Does it mean that we only split group for those tables which are creating ?

Reply:
same as before, we only split the group for the tables which had been created